save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/ParseUrlInTab.kt

```diff
@@ -21,6 +21,16 @@ external interface HasSelectedMenu<T : Enum<T>> : State {
     var selectedMenu: T
 }
 
+/**
+ * The class is needed to store the paths of different tabs in different Views
+ * @property pathDefaultTab (must not start with a "#" for the correct execution of [navigate] in the [navigateToLinkWithSuffix] function)
+ * @property extendedViewPath
+ */
+data class PathsForTabs(
+    val pathDefaultTab: String,
+    val extendedViewPath: String
```

Comment:
maybe add a custom constructor with
```kotlin
extendedViewPath = "#$pathDefaultTab"
```

Reply:
No, these elements are not so interconnected,
by pathDefaultTab I meant the path for the default tab, and extended View Path is the prefix of the path for the rest of the tab (there is usually some word added at the beginning, for example, Project or Organization).

But I realized that it is not clear from the name, now I will add KDoc